spec/forms/import_form_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'rails_helper'
+
+RSpec.describe ImportForm do
+  let(:url) { "http://example.com" }
+  let(:preview) { "0" }
+  let(:form) { ImportForm.new(:url => url, :preview => preview) }
+  let(:rdf_importer) { instance_double("RdfImporter") }
+  let(:term_list) { instance_double("ImportableTermList") }
+
+  before do
+    allow(RdfImporter).to receive(:new).and_return(rdf_importer)
+    allow(rdf_importer).to receive(:term_list).and_return(term_list)
+  end
+
+  describe ".new" do
+    it "should create an RdfImporter" do
+      expect(RdfImporter).to receive(:new).and_return(rdf_importer)
+      form
+    end
+  end
+
+  describe "#valid?" do
+    it "should return the state of errors.empty?" do
```

Comment:
Stubbing methods on the object under test is usually dangerous. Is this an implementation detail?